### DISCUSSION TITLE
Developer meeting 2nd of October - Demo-driven Development

### DISCUSSION THREAD

**Discussion by knmcguire** [bitcraze/discussions#1565] [Announcements]
We will be restarting our developer meetings! This time we will be talking about the importance of demo-driven development at Bitcraze and a bit of the history of how it build us as a company. 

Date: Wednesday 2nd of October
Time: 15:00 (3 pm) Central European Summer time ([Timebuddy timezone reference](https://www.worldtimebuddy.com/?qm=1&lid=2673730,1850147,5368361&h=2673730&date=2024-10-2&sln=15-16&hf=1))
Meeting link: Meeting is done! [See the recording here](https://youtu.be/ebxdJgpx7kE).
Speakers: @knmcguire @ataffanel 
Slides: [demo_driven_development_devmeeting_2024.pdf](https://github.com/user-attachments/files/17241256/demo_driven_development_devmeeting_2024.pdf)

You'll need to crash a couple of times before you can fly, but it will make you achieve success faster :D We will tell you about the importance of why it is okay to fail some times.

![dev meeting 02october 2024](https://github.com/user-attachments/assets/4b257b0f-c893-4040-baab-cf4834803654)

Indicate if you want to join by 👍

**Comment by matejkarasek** [comment]
Nice, will try to join!
PS: The text has a wrong date (October **_3_**), might cause some confusion, though from the rest it seems clear it is on Wednesday...

**Reply by knmcguire** [reply]
Updated! Thanks 😄

**Comment by knmcguire** [comment]
Yesterday's blogpost was about this topic! Mostly looking at demos we have showed at fairs: 

https://www.bitcraze.io/2024/09/bitcraze-and-demo-driven-development/

**Comment by knmcguire** [comment]
The meeting was done, it was a lot of fun! 

If you weren't there, we got the recording and slides right here for you

https://youtu.be/ebxdJgpx7kE 
[demo_driven_development_devmeeting_2024.pdf](https://github.com/user-attachments/files/17241268/demo_driven_development_devmeeting_2024.pdf)
